### ISSUE TITLE
themes aren't global

### ISSUE THREAD

**Issue by Payback80** [yhat/rodeo#666]
changing the theme only change the theme only in the editor window

**Comment by Uguudei** [comment]
Right. There is an open thread from almost two years ago. And it hasn't been resolved yet. I don't think it will be fixed anytime soon. I am currently trying Pycharm pro now.
